entries/h/hpe.com.json

```diff
@@ -0,0 +1,20 @@
+{
+  "HPE": {
```

Comment:
```suggestion
  "Hewlett Packard Enterprise": {
```
I would use the full business name as 'HPE' is more of an "also know as" name

Reply:
Thanks! 😃